frontend/packages/db-structure/src/parser/sql/postgresql/converter.ts

```diff
@@ -219,33 +219,54 @@ export const convertToDBStructure = (ast: RawStmtWrapper[]): DBStructure => {
   }
 
   function handleCommentStmt(commentStmt: CommentStmt) {
-    if (commentStmt.objtype !== 'OBJECT_TABLE') return
+    if (
+      commentStmt.objtype !== 'OBJECT_TABLE' &&
+      commentStmt.objtype !== 'OBJECT_COLUMN'
+    )
+      return
     const objectNode = commentStmt.object
     if (!objectNode) return
-
     const isList = (stmt: Node): stmt is { List: List } => 'List' in stmt
     if (!isList(objectNode)) return
 
-    // Handles statements like `COMMENT ON TABLE <table_name> IS '<comment>';`.
-    // NOTE: PostgreSQL allows only one comment to be added to one table per statement,
-    // so we can reasonably assume the number of `<table_name>` elements is 1.
-    const item = objectNode.List?.items?.[0]
-    if (!item) return
-    const tableName =
+    const comment = commentStmt.comment
+    if (!comment) return
+
+    // `COMMENT ON TABLE <table_name> IS '<comment>';`
+    // or
+    // `COMMENT ON COLUMN <table_name>.<column_name> IS '<comment>';`.
+    // e1 is the table name, e2 is the column name. e2 may be undefined.
```

Comment:
Is it true?

e1 could be schema name in some cases💭

```sql
COMMENT ON TABLE my_schema.my_table IS 'Employee Information';
```

https://www.postgresql.org/docs/current/sql-comment.html

Reply:
Oh, sorry for the misunderstanding!

Reply:
@sasamuku 

Thank you. As you pointed out, I’ve made adjustments to ensure it works correctly even in cases where the schema name is included in the object value. However, I haven’t added any tests yet. How about this one?https://github.com/liam-hq/liam/pull/162/commits/2cdbcfb7933631065d1ee8b4f9a422f7bb32f65c

I think that if we want to ensure support for `schema names` in tests, we should also include support for `schema names` in other `CREATE TABLE` test cases.

Reply:
Great!
That's ok to not support schema name.
Because we don't have schema names in `DBStructure` and no need for now 👍